lab_1_keywords_tfidf/main.py

```diff
@@ -99,7 +156,21 @@ def calculate_tfidf(term_freq: dict[str, float], idf: dict[str, float]) -> Optio
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if term_freq and isinstance(term_freq, dict) and isinstance(idf, dict):
```

Comment:
revert the condition and return None immediately

Reply:
Done